AbstractModels/AbstractUser.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Model object generated by: Skipper (http://www.skipper18.com)
+ * Do not modify this file manually.
+ */
+
+namespace App\Models\AbstractModels;
```

Comment:
Namespace changed from Base to AbstractModels (naming in-line with PHP internals conventions)